src/components/FooterLogo.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import { withClient } from 'cozy-client'
+
+export class FooterLogo extends React.Component {
+  state = {
+    logos: []
+  }
+
+  async componentDidMount() {
+    const logos = await this.fetchHomeLogos()
+    this.setState({ logos })
+  }
+
+  async fetchHomeLogos() {
+    try {
+      const context = await this.props.client
+        .getStackClient()
+        .fetchJSON('GET', '/settings/context')
+      const logos = get(context, 'data.attributes.home_logos', [])
+      return logos
+    } catch (error) {
+      return []
+    }
+  }
```

Comment:
Nit: I think it's better to have the method completely extracted from the component, it makes it more movable/shareable.

```
const fetchHomeLogos = client => {
  ... 
}

async fetchHomeLogos() {
  return fetchHomeLogos(this.props.client)
}
```

Also here, we will fetch the context every time which might induce a flickering.